curation-migration-datasync/database/dataService.ts

```diff
@@ -130,6 +132,78 @@ export class DataService {
     });
   }
 
+  /**
+   * updates the curated_feed_prospects table and curated_feed_queued_items
+   * fields, if they are set in the eventBody.
+   * won't update the fields if they are set to null in the eventBody.
+   * @param eventBody event body
+   * @param curatedRecId curatedRecId corresponding to the approvedItem's externalId
+   * @param domainId domain id from parser, optional field set only when
+   *        publisher is not null in the event body.
+   */
+  public async updateApprovedItem(
+    eventBody: ApprovedItemPayload,
+    curatedRecId: number
+  ): Promise<void> {
+    const item = await this.db(config.tables.curatedFeedItems)
```

Comment:
We don't necessarily need this record (since we can do a partial update), but if we keep this flow -- should probably throw an error if `first` doesn't return any result

Reply:
ya will throw an error/return based on what we decide here: https://github.com/Pocket/curation-tools-data-sync/pull/123#discussion_r843169161

Reply:
throwing error here, and logging error in the calling function in eventConsumer